app/views/incomeTaxComparison/IncomeSummary.scala.html

```diff
@@ -64,22 +64,22 @@ <h2 id="incomeSummaryHeading" class="heading-large">@{incomeSummaryHeading}</h2>
         @for((employmentIncomeSourceDetail, count) <- incomeSourceViewModel.employmentIncomeSourceDetail.zipWithIndex) {
 
             <tr>
-                <td scope="row" role="gridcell" class="table-title">
-                    <span class="table-heading" aria-hidden="true" id="employerName">
+                <td role="cell" class="table-title">
```

Comment:
Guessing this is related to the role=grid. You probably don't need to add role=cell